pygrocy/grocy_api_client.py

```diff
@@ -334,6 +336,12 @@ def get_chore(self, chore_id: int) -> ChoreDetailsResponse:
         return ChoreDetailsResponse(parsed_json)
 
     def execute_chore(self, chore_id: int, done_by: int = None, tracked_time: datetime = datetime.now()):
+
+        # Grocy API expects UTC time; time returned from datetime.now() is local time without timezone
+        # information, so timezone information must be attached.
+        local_tz = get_localzone()
+        tracked_time = local_tz.localize(tracked_time)
```

Comment:
this is shadowing the parameter `tracked_time`. Could you use a different name like `localized_tracked_time`?

Reply:
No problem, have done.

Reply:
No problem, have done.